portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsJumpToPage.testcase

```diff
@@ -318,11 +318,20 @@ definition {
 				FormFields.viewTextField(
 					fieldLabel = "Destiny ${option}",
 					fieldName = "${fieldNameDestiny2}");
+
+				FormPortlet.previousPage();
+				FormPortlet.waitFormRenderer(text = "Next");
 			}
 			else {
 				FormFields.viewTextField(
 					fieldLabel = "Destiny ${option}",
 					fieldName = "${fieldNameDestiny3}");
+
+				FormPortlet.previousPage();
+				FormPortlet.waitFormRenderer(text = "Next");
+
+				FormPortlet.previousPage();
+				FormPortlet.waitFormRenderer(text = "Next");
```

Comment:
@linolaoj I didn’t understand these changes. Can you please explain? Thanks

Reply:
@carolmariaabb , this fix changes the behavior of the "Previous" button because it is filtering the rules. The user will need to press the "Previous" more times to go back. Please let me know if this should be different.

Reply:
@linolaoj What do you mean by `will need to press the "Previous" more times to go back`? If the user clicks Previous once, will he be stuck on the same page until he clicks again?

Reply:
@carolmariaabb , if the user clicks the "Previous" once he will go back one page. The user won't be stuck. Before the fix, the user could come back multiple pages only clicking "Previous" once, because all the rules were evaluated. Please let me know if more details are needed.

Reply:
Hi @linolaoj, Now I get it. Thanks!.

However, as we are changing the expected behavior, I believe we should confirm with the Forms Team the expected behavior in this case. In my opinion, if the user skips a page by clicking the `Next` button, when clicking the `Previous` button he must skip the same page(s). Does it make sense to you?

I read the comment on this [PTR](https://issues.liferay.com/browse/PTR-2590) and it was clear to me that the rules of skipped pages should be ignored, right?
However, it's not clear to me what behavior is expected when clicking the Previous button on a page that is a result of the `Jump to Page` rule.

Could you please confirm (it could be in the PTR mentioned above) the expected behavior for this scenario?

Thanks.

Reply:
Hi @carolmariaabb , I've asked on [PTR](https://issues.liferay.com/browse/PTR-2590?focusedCommentId=2504346&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2504346). Thanks!